Extension/src/LanguageServer/Providers/codeActionProvider.ts

```diff
@@ -97,11 +98,20 @@ export class CodeActionProvider implements vscode.CodeActionProvider {
             let wsEdit: vscode.WorkspaceEdit | undefined;
             let codeActionKind: vscode.CodeActionKind = vscode.CodeActionKind.QuickFix;
             if (command.edit) {
-                // Inline macro feature.
-                codeActionKind = CodeActionProvider.inlineMacroKind;
                 wsEdit = new vscode.WorkspaceEdit();
+                if (command.command === 'C_Cpp.AddMissingInclude') {
+                    if (os.platform() === 'win32') {
+                        command.edit.newText = command.edit.newText + "\r\n";
```

Comment:
This isn't conditional isn't necessary. VS Code will convert `\n` to `\r\n` for CLRF files on Windows. We have other code that just adds `\n` such as `src\LanguageServer\Providers\documentFormattingEditProvider.ts`.